tests/mount_test.c

```diff
@@ -120,14 +120,14 @@ Suite* make_mount_suite(void) {
 	return s;
 }
 
-gboolean enable_debug = true;
 
```

Comment:
If I were being picky ;-), then I'd note that you have left an extra blank line in all the places you deleted this gboolean :-)  I'll let @jodh-intel comment on if that irks him or not...